spec/document_builder_example_spec.rb

```diff
@@ -40,6 +40,24 @@
       expect(parsed_sample_xlsx_file.worksheets[0]
                  .rows[6].cells[0].raw_text).to match(/.*ONLYOFFICE Commercial director.*John Smith.*/)
     end
+
+    it 'create document with custom data works' do
+      @introduction_page.input_name_company_position
+      sample_docx_custom_file = @introduction_page.create_docx_from_sample_data
+      parsed_sample_custom_docx_file = OoxmlParser::Parser.parse(sample_docx_custom_file)
+      expect(parsed_sample_custom_docx_file.elements[1]
+                 .character_style_array[0].text).to include('Ivan Petrov')
```

Comment:
This text should extracted to contstants in `DocumentBuilder` and called like `DocumentBuilder::DEFAULT_NAME`
Because if by any reason this name should be changed - it'll need to be changed in several places, not once